style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  background-color: #3c3a39;
+  padding: 20px;
+}
+
+.navbar-wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navbar-logo {
+  text-decoration: none;
+  font-size: 20px;
+  color: white;
+  font-family: "Inter", sans-serif;
+}
+
+.nav-links {
+  list-style-type: none;
+}
+
+.nav-link {
+  display: none;
+}
+
+.menu {
+  width: 30px;
+  filter: invert(100%);
+}
+
+.headline-section {
+  background-color: #131110;
+  width: 100%;
+  height: 600px;
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  justify-content: center;
+  align-items: center;
+}
+
+.headline-section-wrapper {
+  padding: 10px;
+}
+
+.headline-title {
+  color: #f55800;
+  font-size: 50px;
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+}
+
+.headline-professional {
+  color: white;
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  margin-bottom: 10px;
+}
```

Comment:
- [ ] The correct `font-size` for this element is `24px` and not `32px`. Kindly use the correct `font-size` so that your design looks identical to that of the Figma template that you chose. 👍 😎

###  Your Design

![image](https://user-images.githubusercontent.com/77106962/234967606-91d85208-0ee8-4985-9634-f0d26e7efd5c.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/234967736-3a15ed61-d6b9-4266-aeef-511824450fd5.png)